src/hooks/UrqlUseQuery.rei

```diff
@@ -1,22 +1,15 @@
-open UrqlTypes;
-
-type partialOperationContextFn =
+type executeQuery =
```

Comment:
Isn't this a breaking change? Since it was exposed to the public, someone could be using it and the next release would inadvertently break their code

Reply:
I suppose that's true. They'd need to be referencing the `type` directly, but like you said that's technically part of the public API. I think we should probably release this as 1.3.0 anyway with a notice about `bs-platform@6.2.1` support. It is an intended change as I think it gives better context around what that function actually is (the `executeQuery` function with support for altering the query execution environment).

Reply:
Ok, sounds good to me!